LetsDoStuff.Domain/Models/User.cs

```diff
@@ -6,6 +6,12 @@ public class User
 
         public string Name { get; set; }
 
+        public string Login { get; set; }
+
+        public string Passeword { get; set; }
```

Comment:
The "Password"; Write the names of variables correctly, please.

Reply:
Sorry, my bad :(